libsnark/zk_proof_systems/plonk/srs.tcc

```diff
@@ -96,19 +96,19 @@ transcript_hasher<ppT>::transcript_hasher(std::vector<uint8_t> &buffer)
     };
 }
 
-/// clear the buffer (for now only for testing)
+// clear the buffer (for now only for testing)
 template<typename ppT> void transcript_hasher<ppT>::buffer_clear()
 {
     this->buffer.clear();
 }
 
-/// get buffer size
+// get buffer size
```

Comment:
Again - the comment makes sense as a doc comment, so let's put it in the header instead.

Reply:
So, if someone wants to know *what* a class or method does, they look in the .hpp (at the interface and doc comments).  If they want to know *how* it does it, they look in the .tcc (at the implementation itself, and comments inside the implementation).

Reply:
Addressed across all files.